docs/src/ToricVarieties/NormalToricVarieties.md

```diff
@@ -45,36 +45,32 @@ For the notation we rely on [Chr91](@cite) and [Ste91](@cite).
 
 ```@docs
 CyclicQuotientSingularity(n::fmpz, q::fmpz)
-continued_fraction_hirzebruch_jung(cqs::CyclicQuotientSingularity)
-continued_fraction_hirzebruch_jung_to_rational(v::Vector{fmpz})
-dual_continued_fraction_hirzebruch_jung(cqs::CyclicQuotientSingularity)
-rational_to_continued_fraction_hirzebruch_jung(r::fmpq)
```

Comment:
Are you sure that it makes the docs better to move these things elsewhere? Now the warning doesn't make sense at this place.

Reply:
We could just have a dedicated page for CyclicCuotientSingularity apart from the more general stuff.

Reply:
Yes, I agree. A dedicated page would be great!